modules/Forum/language/French/language.php

```diff
@@ -0,0 +1,156 @@
+	'spam_wait' => 'Merci de patienter {x} secondes avant de reposter.',
+	'overview' => 'Vue d\'ensemble',
+	'no_label' => 'No label',
+	
+	// Homepage
+	'latest_announcements' => 'Dernières annonces',
+	'read_full_post' => 'Lire en entier',
+	
+	// Admin tab
+	'labels' => 'Étiquettes',
+	'new_forum' => '<i class="fa fa-plus-circle"></i> Nouveau forum',
+	'new_label' => '<i class="fa fa-plus-circle"></i> Nouvelle étiquette',
+	'new_label_type' => '<i class="fa fa-plus-circle"></i> Nouveau type d\'étiquette',
+	'label_types' => 'Types d\'étiquette',
+	'creating_label' => 'Création d\'un nouvelle étiquette',
+	'creating_label_type' => 'Création d\'un nouveau type d\'étiquette',
+	'editing_label' => 'Modification de l\'étiquette',
+	'editing_label_type' => 'Modification du type d\'étiquette',
+	'label_name' => 'Nom de l\'étiquette',
+	'label_type' => 'Type de l\'étiquette',
+	'label_type_name' => 'Nom du type d\'étiquette',
+	'label_type_html' => 'HTML du type d\'étiquette',
+	'label_type_html_help' => 'L\'HTML doit inclure {x} qui sera par la suite remplacer par le nom de l\'étiquette',
+	'label' => 'Étiquette',
+	'label_forums' => 'Étiquette du forum',
+	'no_forums' => 'Aucun forum',
+	'no_labels_defined' => 'Aucune étiquettes n\'ont été définie.', 
+	'no_label_types_defined' => 'Aucun type d\'étiquette n\'ont été définie.',
+	'label_edit_success' => 'Étiquette modifié.',
+	'label_type_edit_success' => 'Tpye d\'étiquette modifié.',
+	'label_creation_success' => 'Étiquette créée.',
+	'label_type_creation_success' => 'Tpye d\'étiquette créé.',
+	'label_creation_error' => 'Il y a eu une erreur lors de la création du type d\'étiquette. Merci de vous assurer que le nom fait moins de 32 charactères et que et que vous avez spécifier un type.',
+	'label_type_creation_error' => 'Il y a eu une erreur lors de la création du type d\'étiquette. Merci de vous assurer que le nom fait moins de 32 charactères et que le HTML moins de 64.',
+	'creating_forum' => 'Création d\'un nouveau forum',
+	'forum_name' => 'Nom du forum',
+	'forum_description' => 'Description du forum',
+	
+	'delete_forum' => 'Suprimmer le forum',
+	'move_topics_and_posts_to' => 'Déplacer les sujets et les messages vers',
+	'delete_topics_and_posts' => 'Suprimmer les sujets et les messages',
+	'forum_permissions' => 'Permissions du forum',
+	'select_a_parent_forum' => 'Séléctionner un forum parent',
+	'parent_forum' => 'Forum parent',
+	'has_no_parent' => 'Aucun parent',
+	'guests' => 'Invités',
+	'group' => 'Groupe',
+	'can_view_forum' => 'Peut voir le forum ?',
+	'can_view_other_topics' => 'Peut voir les sujets des autres joueurs ?',
+	'can_create_topic' => 'Peut créer des sujets',
+	'can_post_reply' => 'Peut répondre ?',
+	'can_moderate_forum' => 'Peut modérer ?',
+	'display_topics_as_news' => 'Mettre les sujet sur la page d\'accueil ?',
+	'forum_created_successfully' => 'Forum créé !.',
+	'forum_layout' => 'Mise en page du forum',
+	'table_view' => 'Voir les tables',
```

Comment:
i haven't saw it on panel so it could not be able to do the best translation for this